### PR TITLE
Setup views

### PR DESCRIPTION
**Pull request by ahmedalmi95**
install express handlebars
require and used express handlebars in app.js
#39

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -8,6 +9,10 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.disable('x-powered-by');
 app.use(cookieParser());
+
+app.engine('handlebars', exphbs({ defaultLayout: 'main' }));
+app.set('view engine', 'handlebars');
+
```
Comment (Fatmasiam) [inline src/app.js]: white space?

File: src/app.js [hunk 2]
```diff
@@ -8,6 +9,10 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.disable('x-powered-by');
 app.use(cookieParser());
+
+app.engine('handlebars', exphbs({ defaultLayout: 'main' }));
```
Comment (Angham116) [inline src/app.js]: complete it with all views folders.

Comment (ahmedalmi95) [reply src/app.js]: done

File: src/app.js (hunk 1 above)
Comment (ahmedalmi95) [reply src/app.js]: done

File: src/app.js [hunk 3]
```diff
@@ -8,6 +10,15 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.disable('x-powered-by');
 app.use(cookieParser());
+app.engine('handlebars', exphbs({ defaultLayout: 'main' }));
```
Comment (Fatmasiam) [inline src/app.js]: remove this? why you put it .. 
whats different between it and lines from 15 to 20

**Review by Angham116 — APPROVED** [review]

File: src/app.js (hunk 3 above)
Comment (ahmedalmi95) [reply src/app.js]: done

**Review by Fatmasiam — APPROVED** [review]

**Review by Anies12 — APPROVED** [review]

File: src/app.js [hunk 4]
```diff
@@ -1,13 +1,24 @@
 const express = require('express');
 const cookieParser = require('cookie-parser');
+const exphbs = require('express-handlebars');
+const { join } = require('path');
+const router = require('./controllers/index');
```
Comment (kannanhassouna) [inline src/app.js:5]: ``` router ```  should not be included in this pull request.
